### PR TITLE
Update chroma goldmark-highlighting

### LINKED ISSUE #6596: Additional empty line in syntax-highlighted code fences when using Windows line endings
### What version of Hugo are you using (`hugo version`)?

<pre>
$ hugo version
Hugo Static Site Generator v0.60.0/extended darwin/amd64 BuildDate: unknown
</pre>

### Does this issue reproduce with the latest release?

Yes.

### How to reproduce

1. Create a new document with a code block using synatx highlighting, e.g.

    ```markdown
        ```python
        # Launch the program.
        def main():
            pass
        ```
    ```

2. Make sure this file is stored using Windows line endings.
3. Generate HTML

### Expected behaviour

HTML should be rendered which looks something like

```python
# Launch the program.
def main():
    pass
```

### Observed behaviour

When using Windows line endings, an additional empty line is printed -- but only for the comment! I.e. I see

```python
# Launch the program

def main():
    pass
```

### Additional notes

It appears that when using Windows line endings, *any* comment in syntax-highlighted code fences (at least for Python and Perl) gets an additional empty line appended.

### PR DESCRIPTION
**Pull request by satotake**
Upstream repositories were fixed
- https://github.com/yuin/goldmark-highlighting/pull/14
- https://github.com/alecthomas/chroma/pull/336

Fixes #6596

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -5,7 +5,7 @@ require (
 	github.com/BurntSushi/toml v0.3.1
 	github.com/PuerkitoBio/purell v1.1.1
 	github.com/PuerkitoBio/urlesc v0.0.0-20170810143723-de5bf2ad4578 // indirect
-	github.com/alecthomas/chroma v0.7.1
+	github.com/alecthomas/chroma v0.7.2-0.20200305040604-4f3623dce67a
```
Comment (bep) [inline go.mod]: I don't want to add non-release versions unless I really have to.

Comment (satotake) [reply go.mod]: OK. Let's wait for releasing it.
Should I close this and recreate then, or leave and update it?

Comment (bep) [reply go.mod]: Up to you.

Comment (satotake) [reply go.mod]: Ok. 
Although I don't have specific reasons, I will close this for now

**Pull request closed without merging**